benchmarks/multiplatform/benchmarks/src/commonMain/kotlin/MeasureComposable.kt

```diff
@@ -91,9 +94,9 @@ suspend fun measureComposable(
             val start = markNow()
 
             repeat(frameCount) {
-                val frameStart = start + nextVSync
+                val frameStart = markNow()
```

Comment:
From https://github.com/JetBrains/compose-multiplatform/pull/5216#discussion_r1930973719:

> We mark frameStart from nextVSync intentionally to catch possible GC. Consider revert this

Reply:
Is it by design that we also include the time of when we wait for vsync?

Originally I noticed it when I compared it to the average frame time, to understand how much frames deviates from it. Having additional time doesn't allow this comparison.

Reply:
>Is it by design that we also include the time of when we wait for vsync?

Yes, It is by design of frames metrics that is greatly affected by GC: we need to count GC in fames measurements (because the benchmarks are also used for GC tuning) and if we count a frame from markNow we will loose GC that could happen after nextVSync and before actual work for a frame.

In general, this PR is very doubtable, we need to discuss it.

Reply:
Reverted to reduce the review scope